packages/utils/src/structs/checkpoint.cairo

```diff
@@ -99,12 +99,12 @@ pub impl TraceImpl of TraceTrait {
         }
     }
 
-    /// Returns the number of checkpoints.
+    /// Returns the total number of checkpoints.
     fn length(self: StoragePath<Trace>) -> u64 {
         self.checkpoints.len()
     }
 
-    /// Returns the checkpoint at given position.
```

Comment:
I think position is easier to read here, maybe we can update the param name?

Reply:
Mmm I honestly like `pos` as the param name—unless you prefer a new name? I'll switch the comment back. Looking at it now, I agree position is easier to read